Formula/cuba.rb

```diff
@@ -0,0 +1,24 @@
+class Cuba < Formula
+  desc "Library for multidimensional numerical integration"
+  homepage "http://www.feynarts.de/cuba/"
+  url "http://www.feynarts.de/cuba/Cuba-4.2.tar.gz"
+  sha256 "6b75bb8146ae6fb7da8ebb72ab7502ecd73920efc3ff77a69a656db9530a5eef"
+
+  option "without-test", "Skip build-time tests (not recommended)"
+
+  def install
+    ENV.deparallelize
+
+    system "./configure", "--prefix=#{prefix}", "--datadir=#{pkgshare}/doc"
```

Comment:
Why is `data` placed in `doc` subdirectory? What would be the default directory?

Reply:
I just verified that `"--datadir=#{pkgshare}/doc"` is redundant, I'll remove that